dist/index.js

```diff
@@ -156,19 +148,7 @@ async function runScan({
 
   failBuild = failBuild.toLowerCase() === "true";
 
-  acsReportEnable = acsReportEnable.toLowerCase() === "true";
-
-  if (outputFormat !== "sarif" && acsReportEnable) {
-    throw new Error(
-      `Invalid output-format selected. If acs-report-enabled is true (which is the default if it is omitted), the output-format parameter must be sarif or must be omitted`
-    );
-  }
-
-  if (acsReportEnable) {
-    cmdArgs.push("-o", "sarif");
-  } else {
-    cmdArgs.push("-o", outputFormat);
-  }
+  cmdArgs.push("-o", outputFormat);
```

Comment:
Nice, this makes it much easier to read and simplifies everything.

Reply:
It hasn't been updated since 2020, I didn't even notice it was in this repo until the search/replace of `acs-report-enable` for this PR!